content/articles/stream-filter-in-java-using-lambda-expressions/index.md

```diff
@@ -88,21 +95,24 @@ List<Employee> georgeWith90MarksAndAbove = employees
   assertThis(georgeWith90MarksAndAbove).hasSize(01);
   assertThis(georgeWith90MarksAndAbove).contains(george);
 ```
-**Explanation**
+#### Explanation
+
 We used multiple conditions with filter() such as marks and identity of the employees
 
 ### Methods for dealing with exceptions
-In order to assess predicates that do not throw exceptions, we have been utilizing the filter up to this point in time. The Java programming language's functional interfaces do not define whatever checked or unchecked exceptions, and this is a good thing.
 
-The handling of exceptions in lambda expressions shall next be explored in detail using a number of alternative approaches.
+To assess predicates that do not throw exceptions, we have been utilizing the filter up to this point in time. Java's functional interfaces do not define whatever checked or unchecked exceptions are thrown, and this is a good thing.

```

Comment:
Very vague why you said it is a good thing. I would add an explanation as to why.